packages/core/core/src/requests/AssetGraphRequest.js

```diff
@@ -396,6 +400,7 @@ export class AssetGraphBuilder {
           depUsedSymbolsDown.clear();
         }
         if (!equalSet(depUsedSymbolsDownOld, depUsedSymbolsDown)) {
+          changedDeps.add(dep);
```

Comment:
Not sure that we actually need to track changes to `usedSymbolsDown`, because outgoing edge orders are probably deterministic, and we are also only sorting the `usedSymbolsUp`.

Reply:
Right, I think this case isn't needed.

> because outgoing edge orders are probably deterministic

(No, they aren't: https://github.com/parcel-bundler/parcel/issues/8038. But AFAICT this could be fixed by using `asset.dependencies` as opposed to `getNodesConnectedFrom(asset)`)